src/styles/stars.css

```diff
@@ -0,0 +1,28 @@
+.card__stars {
```

Comment:
```suggestion
.stars {
```

Reply:
Hi ! ok, help me. First I add Roboto via @import, than i changed it like there (https://www.codegrepper.com/code-examples/css/css+roboto+font)  in <head> and html. How to do it correct? thanks

Reply:
It's better you use link if you are loading it from CDN or google fonts. If the font is in your local directory then loading it using import or link will not have too much significant difference. But if you are loading it using a third party then it's always better to use link .

If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review 🙂